ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -476,6 +488,32 @@ export default function ViewQuote() {
             onQuoteDetailsIsOpened={quoteDetailsOpened}
           />
         )}
+        {shouldShowPriceDifferenceWarning && (
+          <div
+            className={classnames(
+              'view-quote__price-difference-warning-wrapper',
+              {
+                high: usedQuote?.priceSlippage?.bucket === 'high',
+                medium: usedQuote?.priceSlippage?.bucket === 'medium',
+              },
+            )}
+          >
+            <ActionableMessage
+              message={
+                <div className="view-quote__price-difference-warning-contents">
+                  {t('swapPriceDifference', priceDifferencePercentage)}
```

Comment:
The second parameter of `t` needs to be an array

Reply:
`t('swapPriceDifference', [priceDifferencePercentage])` will correctly pass `priceDifferencePercentage` to the translation helper to be substituted in the message